pkg/controller/migplan/registry_health_running.go

```diff
@@ -115,21 +115,10 @@ func (r *registryHealth) run() {
 				case isSrcRegistryPodUnhealthy && plan.Status.HasCondition(RegistriesHealthy):
 					r.enqueue(plan)
 					continue
-
-				//enqueue a reconcile event when the src registry pod is healthy and the plan is not ready
-				case !isSrcRegistryPodUnhealthy && !plan.Status.HasCondition(RegistriesHealthy):
-					r.enqueue(plan)
-					continue
-
 				//enqueue a reconcile event when the destination registry pod is unhealthy and the plan is ready
 				case isDestRegistryPodUnhealthy && plan.Status.HasCondition(RegistriesHealthy):
```

Comment:
you might need to add another condition in with `&&` operator here, see previous comment